src/lib.rs

```diff
@@ -340,7 +341,7 @@ where
             .expect("trying to resize the hidden frame.");
 
         decorations.resize(width.get(), height.get());
-        self.buttons.arrange(width.get());
+        self.buttons.arrange(width.get(), get_margin_h_lp(&self.state));
```

Comment:
Looks like CI is not happy about this formatting:
```suggestion
        self.buttons
            .arrange(width.get(), get_margin_h_lp(&self.state));
```